models/show.js

```diff
@@ -0,0 +1,18 @@
+import mongoose from 'mongoose'
+
+const NetflixShow = mongoose.model('NetflixShow', {
+  show_id: { type: Number },
+  title: { type: String },
+  director: { type: String },
+  cast: { type: String },
+  country: { type: String },
+  date_added: { type: String },
+  release_year: { type: Number },
+  rating: { type: String },
+  duration: { type: String },
+  listed_in: { type: String },
+  description: { type: String },
+  type: { type: String },
```

Comment:
Nice that you extracted this to its own file! If you just want to specify the type, you could write it in one line:
```
show_id: Number,
title: String,
```
etcetera, and if you have more attributes, you write like this:
```
show_id: {
  type: Number,
  unique: true,
},
title: {
  type: String
  required: true,
},

```